sklearn/metrics/tests/test_common.py

```diff
@@ -1276,3 +1280,26 @@ def test_thresholded_multilabel_multioutput_permutations_invariance(name):
 
         current_score = metric(y_true_perm, y_score_perm)
         assert_almost_equal(score, current_score)
+
+
+@pytest.mark.parametrize(
+    'name',
+    set(THRESHOLDED_METRICS) - METRIC_UNDEFINED_BINARY_MULTICLASS)
+def test_thresdhold_metric_is_permutation_invariance(name):
```

Comment:
the grammar here is weird `test_thresdhold_metric_is_permutation_invariant` perhaps or `test_thresdhold_metric_is_invariant_to_permutation` or `test_thresdhold_metric_permutation_invariance`.